pkg/virt-handler/device-manager/generic_device.go

```diff
@@ -261,6 +262,16 @@ func (dpi *GenericDevicePlugin) Allocate(ctx context.Context, r *pluginapi.Alloc
 	dev.HostPath = dpi.devicePath
 	dev.ContainerPath = dpi.devicePath
 	dev.Permissions = "rw"
+	if cgroups.IsCgroup2UnifiedMode() {
```

Comment:
I would expect to make this configurable on the GenericDevicePlugin for better testability.

Reply:
I think I can change the implementation and pass the permissions via `NewGenericDevicePlugin`. But then probably it makes sense to propagate them to `NewDeviceController` from `NewController` in `vm.go`. So it will be two levels above in the callstack. What do you think? Is it okay to do like that?